src/options.js

```diff
@@ -1,6 +1,6 @@
 class RecrasOptions {
     static hostnamesDebug = [
-        '172.16.0.2', // Local development
+        'nginx:8886', // Local development
```

Comment:
Not a blocker: It seems weird to me that this library has knowledge about implementation details of the Recras codebase. Shouldn't the 'I know what I'm doing, you can trust me on this' option be in the initialization of the library?

Reply:
How would you suggest we change this?